### ISSUE TITLE
SESSDATA格式改变

### ISSUE THREAD

**Issue by 7rikka** [SocialSisterYi/bilibili-API-collect#742]
变长了

**Comment by cxw620** [comment]
后面多出来的一段估计是追踪风控用的, APP 的 access_key 也多出来一段, AATAA 结尾, Web 就是 AAHAA 结尾. 
这玩意加密数据, 谁知道里面有什么, 估计记录了登录 IP 什么的吧.

**Comment by SessionHu** [comment]
是的, 到现在 SESSDATA 还是很长很长, 不过好像后面一长串有很多是可以截掉的, 不影响请求, 真正有用的只有前面一小段, close?
